prov/verbs/src/fi_verbs.c

```diff
@@ -614,6 +615,11 @@ fi_ibv_getepinfo(const char *node, const char *service,
 	ret = rdma_create_ep(id, rai, NULL, NULL);
 	if (ret) {
 		ret = -errno;
+		if (ret == -ENOENT) {
+			FI_LOG(1, "verbs",
+				"rdma_create_ep() returned ENOENT.  We are likely dealing with a usnic_X device that has rdma_cm bugs, so skip the verbs provider instead.\n");
```

Comment:
I wondered if you would find that objectionable.  I have a bias for writing messages that are more informative rather than less, and I prefer to avoid breaking user-facing string literals to make it easier to grep the source for them.  How about this instead?

```
"rdma_create_ep()-->ENOENT; likely usnic bug, skipping verbs provider.\n"
```

I can also break that into two string literals on separate lines if you prefer to keep the message at a hard 80 column limit.  That would be:

```
"rdma_create_ep()-->ENOENT; likely usnic bug, "
"skipping verbs provider.\n");
```

I prefer the first to the second, but I primarily care that usnic stops getting clobbered by the verbs provider.